pkg/sqlmigrations/migrations.go

```diff
@@ -522,6 +522,15 @@ func runStmtAsRootWithRetry(ctx context.Context, r runner, stmt string) error {
 	return err
 }
 
+// SettingsDefaultOverrides documents the effect of several migrations add an
```

Comment:
I can't figure out what `several migrations add an explicit value...` is trying to say.

Reply:
was missing a `that`. Fixed.